Modules/_testcapimodule.c

```diff
@@ -3369,15 +3367,18 @@ slot_tp_del(PyObject *self)
     /* Save the current exception, if any. */
     PyErr_Fetch(&error_type, &error_value, &error_traceback);
 
+    PyObject *tp_del = PyUnicode_InternFromString("__tp_del__");
+    if (tp_del == NULL) {
+        return;
+    }
```

Comment:
The error should be cleared and the current exception restored, right?

```suggestion
    if (tp_del == NULL) {
        PyErr_WriteUnraisable(NULL);
        PyErr_Restore(error_type, error_value, error_traceback);
        return;
    }
```

Reply:
Right, I forgot push the fix for this yesterday, fixed now.